crates/util/python-utils/src/drt.rs

```diff
@@ -0,0 +1,239 @@
+use std::str::FromStr;
+
+use bdk_wallet::{
+    bitcoin::{
+        consensus::encode::serialize_hex, hashes::Hash, secp256k1::SECP256K1, taproot::LeafVersion,
+        Address, FeeRate, TapNodeHash, Transaction, XOnlyPublicKey,
+    },
+    descriptor::IntoWalletDescriptor,
+    miniscript::{miniscript::Tap, Miniscript},
+    template::DescriptorTemplateOut,
+    KeychainKind, TxOrdering, Wallet,
+};
+use pyo3::prelude::*;
+use reth::primitives::Address as RethAddress;
+
+use crate::{
+    constants::{BRIDGE_IN_AMOUNT, MAGIC_BYTES, NETWORK, RECOVER_DELAY, UNSPENDABLE},
+    error::Error,
+    parse::{parse_el_address, parse_xonly_pk},
+    taproot::{client, sync_wallet, taproot_wallet, ExtractP2trPubkey},
+};
+
+/// Generates a Deposit Request transaction (DRT).
+///
+/// # Arguments
+///
+/// - `el_address`: Execution layer address of the account that will receive the funds.
+/// - `bitcoind_url`: URL of the `bitcoind` instance.
+/// - `bitcoind_user`: Username for the `bitcoind` instance.
+/// - `bitcoind_password`: Password for the `bitcoind` instance.
+///
+/// # Returns
+///
+/// A signed (with the `private_key`) and serialized Deposit Request transaction.
+#[pyfunction]
+pub(crate) fn deposit_request_transaction(
+    el_address: String,
+    musig_bridge_pk: String,
+    bitcoind_url: String,
+    bitcoind_user: String,
+    bitcoind_password: String,
```

Comment:
Maybe we should wrap this up in a `BitcoindCreds` type?

Reply:
Yes, but this is the Python interface, we need to pass strings somehow